src/Http/Productions/CreateProductionValidator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace CultuurNet\UDB3\Http\Productions;
+
+use CultuurNet\Deserializer\DataValidationException;
+use CultuurNet\UDB3\Http\Deserializer\DataValidator\DataValidatorInterface;
+
+class CreateProductionValidator implements DataValidatorInterface
+{
+    public function validate(array $data): void
+    {
+        $messages = [];
+        $messages = array_merge($messages, $this->validateName($data));
+        $messages = array_merge($messages, $this->validateEvents($data));
+
+        if (!empty($messages)) {
+            $exception = new DataValidationException();
+            $exception->setValidationMessages($messages);
+            throw $exception;
+        }
+    }
+
+    private function validateName(array $data): array
+    {
+        if (!isset($data['name'])) {
+            return [
+                ['name' => 'Required but could not be found'],
+            ];
```

Comment:
Shouldn't this be `return ['name' => 'Required but could not be found'];`?
So the errors would be printed in JSON as:
```
{
  name: "Required but could not be found",
  eventIds: "Required but could not be found"
}
```
Instead of
```
[
  {
    name: "Required but could not be found",
  },
  {
    eventIds: "Required but could not be found"
  }
]
```

Reply:
Here's an example from another validator inside a test: https://github.com/cultuurnet/udb3-silex/blob/master/tests/Http/Deserializer/Address/AddressJSONDeserializerTest.php#L34-L41

Reply:
Woah, good spot !